applications/CoSimulationApplication/tests/test_thermal_rom_co_sim.py

```diff
@@ -0,0 +1,101 @@
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+if kratos_utilities.CheckIfApplicationsAvailable("ConvectionDiffusionApplication"):
+    import KratosMultiphysics.ConvectionDiffusionApplication
+
+if kratos_utilities.CheckIfApplicationsAvailable("CoSimulationApplication"):
+    from KratosMultiphysics.CoSimulationApplication.co_simulation_analysis import CoSimulationAnalysis
+
+@KratosUnittest.skipIfApplicationsNotAvailable("ConvectionDiffusionApplication")
+@KratosUnittest.skipIfApplicationsNotAvailable("RomApplication")
+@KratosUnittest.skipIfApplicationsNotAvailable("MappingApplication")
+class TestThermalRomCoSim(KratosUnittest.TestCase):
+
+    def testConvDiffStationaryRom2DCoSim(self):
+        self.work_folder = "thermal_rom_co_sim_test_files/"
+        parameter_file_name = "ProjectParameters_CoSimulation_rom.json"
+
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            # Set up simulation
+            with open(parameter_file_name, 'r') as parameter_file:
+                cosim_parameters = KratosMultiphysics.Parameters(parameter_file.read())
+
+            # Generate output.dat for center domain
+            parameter_file_name_center = "ProjectParameters_center.json"
```

Comment:
please clean aka remove those after the test

maybe add a debug flag or so if needed

Reply:
Done.

Reply:
I meant to remove the files, otherwise they pollute the working dir

The rule is that each test is in charge of cleaning up the files it creates

Reply:
I must admit, I'm a bit lost. It appears that all tests in the CoSimulation application have their corresponding ProjectParameters_{physics}.json files.

Reply:
Ok let me try to explain more clear. When running tests, sometimes output files are generated, e.g. text or vtk files to give some examples. The test should use them e.g. for checking of values/results, and then delete them.
The working directory should remain clean, i.e. `git status` should not show changes after running a test. Otherwise it can get very confusing when one runs many tests, and each of them does not clean up after themselves.

In your case it seems like you write an additional projParams file to the disk, but do not remove it afterwards (not sure though, can you confirm?)

Here is an example:
https://github.com/KratosMultiphysics/Kratos/blob/2a7e8f2e71f004da5dc260dd0d33d92d254c3b26/applications/CoSimulationApplication/tests/co_simulation_test_factory.py#L174-L187
This tests creates a bunch of output files, which are getting deleted afterards

Hopefully this is more clear :(

Reply:
Okay, I see your point. In my case, it just updates the ProjectParameters_center.json and ProjectParameters_outside.json. It does not create a new one. I am doing something inspired from other CoSim tests, for instance:
https://github.com/KratosMultiphysics/Kratos/blob/2a7e8f2e71f004da5dc260dd0d33d92d254c3b26/applications/CoSimulationApplication/tests/test_mok_fsi.py#L114-L180